### ISSUE TITLE
react-static is improperly tagged on NPM

### ISSUE THREAD

**Issue by robbyemmert** [nozzle/react-static#740]
### Expected Outcome:
`npm install -g react-static` should install the latest stable release

### Actual Outcome:
`npm install -g react-static` installs a broken beta build

### Steps to reproduce:
`npm install -g react-static`
`react-static create` currently returns an error (see #713 )
`npm info react-static`
The version reads: react-static@6.0.0-beta.17

### Notes:
Running `npm install x-package-name` should never install a beta build, unless specified with the 'beta' tag (i.e. `npm install x-package-name@beta`).

### Steps to fix:
Point the 'latest' tag on NPM to the latest stable release (currently v5.9.5)

TLDR; https://docs.npmjs.com/getting-started/using-tags under "Publishing with tags"

### Environment

1.  `react-static -v`: error: unknown option `-v'
2.  `node -v`: v10.10.0
3.  `yarn --version or npm -v`: 6.4.1
4.  Operating system: Mac OS High Sierra
5.  Browser and version (if relevant): n/a

**Comment by tannerlinsley** [comment]
Dang. Not sure when that happened. I'll fix this tomorrow. Actually it
probably happened while we were experimenting with semantic-release.

**Comment by tannerlinsley** [comment]
This should be fixed now. http://unpkg.com/react-static@latest/
